src/TraceEvent/Computers/StartStopActivityComputer.cs

```diff
@@ -149,6 +149,13 @@ public StartStopActivityComputer(TraceLogEventSource source, ActivityComputer ta
                     }
                 }
 #endif
+                // Special case - OpenTelemetry-Sdk
+                else if (data.Opcode == TraceEventOpcode.Info && data.providerGuid == OpenTelemetrySdkProvider)
```

Comment:
Lower down there are already special cases specifically for fixing up providers that don't follow the start/stop conventions completely (e.g. FixAndProcessAppInsightsEvents). Should OTel be handled there instead?

Reply:
@pharring Thanks for the ask. That will cause more changes. On line 154 (before change), there's an algorithm to check if it is **maybe** event source, and the `OpenTelemetry-Sdk` event source won't satisfy that check:

```csharp
if (!TraceEventProviders.MaybeAnEventSource(data.ProviderGuid))
{
    return;
}
```

We could update the algo to special case `OpenTelemetry-Sdk` there, and some other, but I don't see a why. It will only end up making it complicated.

Reply:
The OTel SDK Provider GUID _is_ an EventSource and `MaybeAnEventSource` will be true.

Reply:
Yes, you are right. I must oversight something. New iteration pushed.